README.md

```diff
@@ -1,16 +1,16 @@
 # Time Based Deny Authentication Action
 
-[![Build Status](https://travis-ci.org/curityio/time-authentication-action.svg?branch=master)](https://travis-ci.org/curityio/time-authentication-action)
+[![Build Status](https://travis-ci.com/curityio/time-authentication-action.svg?branch=master)](https://travis-ci.com/curityio/time-authentication-action)
```

Comment:
Do we have an account on travis-ci.com? Currently, it looks like the time-authentication-action project does not exist there? Or should I have some permissions to access it?

Reply:
True. I didn't double check that the project doesn't exist on `travis-ci.com`. I assumed it was moved. It exists here: https://travis-ci.org/github/curityio/time-authentication-action but there's a note: 
> [...] the building on [travis-ci.org](http://www.travis-ci.org/) is ceased. Please use [travis-ci.com](http://www.travis-ci.com/) from now on.

For the purpose of this PR, I think, we can remove the build status from the Readme. We don't usually specify that.